pkg/locspec/locations.go

```diff
@@ -504,6 +504,17 @@ func SubstitutePath(path string, rules [][2]string) string {
 		if !strings.HasSuffix(to, separator) {
 			to = to + separator
 		}
+
+		// If 'from' is '.' and path is relative, prepende 'to'.
+		if from == "."+separator && !filepath.IsAbs(path) {
+			return strings.Replace(path, "", to, 1)
+		}
+
+		// If 'to' is '.' and path is absolute. Delete matching `from` if any.
+		if to == "."+separator && filepath.IsAbs(path) {
```

Comment:
I don't understand why this should be the case. Why shouldn't the mapping `non/absolute/path -> .` work?

Reply:
I didn't thought about relative paths mapping use cases. Thanks pointing it out. Changes coming to support it.